scripts/create_tables.sql

```diff
@@ -1 +1,27 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING,
+  dob DATE,
```

Comment:
Given the format of the dates in the CSV file, I don't think that DATE will accept that format. VARCHAR or TEXT would work.